db/db_test2.cc

```diff
@@ -2249,10 +2250,32 @@ TEST_F(DBTest2, DISABLED_GetRaceFlush) {
   // Get() is issued after the first Put(), so it should see either
   // "v1" or "v2".
   ASSERT_NE("NOT_FOUND", Get("foo"));
```

Comment:
Any "v*" is reasonable, as Get() happens in another thread while an update is happening, but not found should not happen.